.github/workflows/integration_tests.yml

```diff
@@ -379,23 +379,14 @@ jobs:
               "$HELM_CHART_NAME" \
               "$HELM_SERVICE_HELM_CHART_NAME" \
               "$JMETER_SERVICE_HELM_CHART_NAME"
-          elif [[ "$USE_RESOURCE_SERVICE" == "true" ]]; then
-            # Use Keptn helm chart to be able to customize the values
-            helm install -n ${KEPTN_NAMESPACE} keptn ${HELM_CHART_NAME} \
-              --create-namespace \
-              --set control-plane.apiGatewayNginx.type=${KEPTN_SERVICE_TYPE} \
-              --set control-plane.resourceService.enabled=true \
-              --set control-plane.resourceService.replicas=1 \
-              --set control-plane.shipyardController.gracePeriod=10 \
-              --set control-plane.shipyardController.preStopHookTime=5 \
-              --set continuous-delivery.enabled=${RUN_CONTINUOUS_DELIVERY_TEST}
           else
             helm install -n ${KEPTN_NAMESPACE} keptn ${HELM_CHART_NAME} \
               --create-namespace \
-              --set control-plane.apiGatewayNginx.type=${KEPTN_SERVICE_TYPE} \
-              --set control-plane.shipyardController.gracePeriod=10 \
-              --set control-plane.shipyardController.preStopHookTime=5 \
-              --set continuous-delivery.enabled=${RUN_CONTINUOUS_DELIVERY_TEST}
+              --set apiGatewayNginx.type=${KEPTN_SERVICE_TYPE} \
+              --set resourceService.enabled=${USE_RESOURCE_SERVICE} \
+              --set shipyardController.gracePeriod=10 \
+              --set shipyardController.preStopHookTime=5 \
+              --set continuousDelivery.enabled=${RUN_CONTINUOUS_DELIVERY_TEST}
```

Comment:
since both resource-svc/non-resource-svc installations use helm now, we can just merge them and set the resource service.enabled value directly